custom/icds_reports/templates/icds_reports/icds_app/download.directive.html

```diff
@@ -7,7 +7,10 @@
     <info-message></info-message>
     <div class="report-content report-content-without-message">
         <div class="alert alert-info" ng-show="$ctrl.showWarning()">
-            <strong>Info!</strong> "You are downloading data for the current calendar month. Data for this report is calculated using calendar months (for example, January 1st-January 31st). Data for this month will not be complete until the end of the month."
+            <strong>Info!</strong> You are downloading data for the current calendar month. Data for this report is calculated using calendar months (for example, January 1st-January 31st). Data for this month will not be complete until the end of the month.
+        </div>
+        <div class="alert alert-info" ng-show="$ctrl.isChildBeneficiaryListSelected()">
```

Comment:
I will offline with saket about this, since i see that it is in the spec, but I am strong against these kind of banners. No one reads them, and they clutter screen, and make it look like the user is doing something wrong when its just a description of the report. Plus the proliferation of them just make users more likely to ignore them, which is problematic because sometimes we use them to indicate action is required as in the option validation ones below.

Reply:
agreed - I also really dislike this UX for the purpose that it serves and think it would be much more suitable to like a small endnote type of thing. let me know if I should still merge or whether you think it'd be better to hash out the experience before getting this in.

Reply:
(going to merge so we can close this out and revisit as a later scope)